support/doc/import-youtube.md

```diff
@@ -0,0 +1,57 @@
+# Import viddeos from Youtube guide
+
+If you already own a Youtube channel, you can use a Peertube scripts to import all your videos inside Peertube.
```

Comment:
- s/scripts/script
- technically, the script works even if you don't own the YouTube channel posting the video(s)

Reply:
I do not know if it worth specifying that, could'nt we face problem if anybody import YT videos they do not own inside Peertube?

Reply:
Then we ought to rephrase: "You can use this script to import YouTube videos inside PeerTube. Be sure you own the videos or have their author's licence to do so."